packages/app-degree-pages/src/core/services/degree-data-prop-resolver-service.jsx

```diff
@@ -27,6 +27,58 @@ const getMajorityOwner = row => {
  *
  */
 // @ts-ignore
+/**
+ * Creates a service object that provides methods to extract and format degree-related data from a row object.
+ * This service acts as a data accessor layer for degree program information, providing a clean API
+ * to retrieve various properties and computed values from raw degree data.
+ *
+ * @param {Object} [row={}] - The raw degree data object containing all degree program information
+ * @param {string} [row.acadPlanMarketingDescription] - Marketing description of the academic plan
+ * @param {string} [row.acadPlanCode] - Academic plan code
+ * @param {string} [row.degreeDescriptionShort] - Short description of the degree
+ * @param {string} [row.acadPlanTypeDescription] - Type description of the academic plan
+ * @param {Array<Object>} [row.majorMapGeneral] - Array of general major maps
+ * @param {string} [row.degreeType] - Type of degree (e.g., "GR" for graduate, "UGCM" for undergraduate)
```

Comment:
Really like the inclusion of these details, especially the descriptive per-field/param details.